src/main/resources/travis/travis.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
You really just mean executables, right? I mean this in a non-cheeky way but e.g. jesConf.tar.enc *is* a binary file but doesn't need to be in an artifact